### PR TITLE
cpu/stm32: fix RTC init and disable outputs

### PR DESCRIPTION
**Pull request by vincent-d**
This PR fixes a delay each time the mcu boots.

If the RTC is powered with a V_bckp, there was ~800ms lost in the RTC counter at each power on.

The RTC alternate function outputs are also disabled, to make PC13 usable as gpio.

### CODE REVIEW

File: cpu/stm32_common/periph/rtc.c [hunk 1]
```diff
@@ -73,31 +73,34 @@ void rtc_init(void)
...
-    RTC->CR |= RTC_CR_TSE;
+        /* Enter RTC Init mode */
+        RTC->ISR = 0;
+        RTC->ISR |= RTC_ISR_INIT;
+        while ((RTC->ISR & RTC_ISR_INITF) == 0) {}
 
-    /* Configure the RTC PRER */
-    RTC->PRER = RTC_SYNC_PRESCALER;
-    RTC->PRER |= (RTC_ASYNC_PRESCALER << 16);
+        /* Set 24-h clock */
```
Comment (aabadie) [inline cpu/stm32_common/periph/rtc.c]: According to the datasheet, this should be done after prescaler configuration.

**Comment by vincent-d** [comment]
> Do you have any advice for testing this

You need a board with Vbackup connected, and do some power down / power up cycles (reboot command is not enough from my experience. You really need to power down).

**Comment by vincent-d** [comment]
> Except the indention

What's wrong with it?

**Comment by haukepetersen** [comment]
nothing, my bad - the diff played a trick no me. So never mind -> ACK

**Comment by vincent-d** [comment]
Closing as #7504 is reworking the driver in a better way

**Pull request closed without merging**